app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class UserMailer < ApplicationMailer
+  default from:  'xw5d@virginia.edu'
+  def jobstatus_email
+    @user = ['xw5d@virginia.edu', 'md5wz@virginia.edu']
```

Comment:
Read these addresses from a config file that is ignored by git so they aren't harvested by spammers on github.